### ISSUE TITLE
Mis-use of internal addresses &Setup

### ISSUE THREAD

**Issue by wvdvegt** [gorco/gf#25]
Hi

I posted an issue (at a2), probably the wrong repo at:

https://github.com/e-ucm/a2/issues/78

and have a question related to it. 

How do we force the setup to use the 'config-test.js' & 'config-example.js' based setup values (and use location as base for generating client side paths instead of internal docker/Linux ip-addresses).

**Comment by manuel-freire** [comment]
This should no longer be an issue - see the closing comment on e-ucm/a2#78

**Comment by wvdvegt** [comment]
Hi Manual,

As stated in the other issue, we reinstalled just before posting this issue and did not see any improvent (see the screenshots). 

We re-installed because we noticed the commit (of then 16 ours old)  of a file solving this problem (by using location to generate the proxy/apipath address, but it does not seem to bet used during the docker install. Hence we still see the 172.\* addresses as seen in the screenshots.

**Comment by manuel-freire** [comment]
An updated docker-compose install was released in [this commit](https://github.com/e-ucm/rage-analytics/commit/62ab59c639613c0097efa7529d283d4cf0e4bcf2); later commits contain relatively minor tweaks - while the underlying bug may have been fixed before this (via the front-end fix), _the updated docker-compose install is a very significant departure from the previous model_, and we have made no attempt to make things work with the old docker-compose setup.

For example, the old setup did not allow for graceful stopping of containers, nor did it retain data across restarts. We are no longer using "linked containers" (which will be deprecated in future docker versions), and have switched over to the new docker networking scheme.
